assets/src/js/admin/onboarding-wizard/components/button/index.js

```diff
@@ -4,21 +4,23 @@ import PropTypes from 'prop-types';
 // Import styles
 import './style.scss';
 
-const Button = ( { onClick, children } ) => {
+const Button = ( { onClick, testId, children } ) => {
```

Comment:
@henryholtgeerts Instead of modifying the component itself, what if we use composition to wrap testable components (which could add the test attribute)?

Reply:
Do you mean to introduce a testable version of components (eg <TestableButton/>) or a generic <Test/> that would wrap the inputs we aim to test? The specific direction I took was for simplicity and to mirror the GIveIO implementation (see their [Checkbox](https://github.com/impress-org/giveio/blob/develop/resources/js/components/ControlCheckBox/index.jsx) component as an example).

Reply:
Yeah... hm.

I was hoping for something like

```
<Testable id="test1">
    <MyComponent />
</Testable>
```

But that doesn't seem to work the way that I had hoped.

Reply:
Yeah, I was just now tryina work out something along those lines locally. Ultimately seems simplest to do the directly applied attribute approach - though I definitely see the appeal in composition here.

Reply:
I think we are on the same page.